### ISSUE TITLE
[Training Issue] Training from backbone

### ISSUE THREAD

**Issue by ryusaeba** [eric612/MobileNet-YOLO#105]
Model : DarkNet53 + YOLOv3

Backbone : DarkNet53

Questions : Have you ever train [darket_yolov3](https://github.com/eric612/MobileNet-YOLO/blob/master/models/darknet_yolov3/yolov3-train.prototxt) before?
 i.e. 
1. Pre-trained weights for backbone DarkNet53
2. Kick off YOLOv3 training
If so, could you please kindly share the solver with me. Thanks

**Comment by eric612** [comment]
I never training start from darknet53 , you can try to modify my [finetune model](https://github.com/eric612/MobileNet-YOLO/tree/master/models/darknet_yolov3) , remember to change learning rate and others params , but it may not get the same mAP with original implementation

**Comment by ryusaeba** [comment]
One more question.
I can get [DarkNet mAP~=54](https://github.com/eric612/MobileNet-YOLO#oringinal-darknet-yolov3) and [Mobilenet mAP~=40.2](https://github.com/eric612/MobileNet-YOLO#performance) by using [DetectionEvaluate layer](https://github.com/eric612/MobileNet-YOLO/blob/master/models/yolov3_coco/mobilenet_yolov3_test.prototxt#L2976-L2989). 
Using coco api will get worse result, have you tried to use it to check mAP results? 
Or this is because DetectionEvaluate is using 11 point and coco api doesn't.

Thanks

**Comment by eric612** [comment]
You can change 11 point to integral in solver.prototxt , and the mAP would be decrease , but this code was copy from [weliu](https://github.com/weiliu89/caffe/tree/ssd) , and so on pelee-ssd and mobilenet-ssd

**Comment by ryusaeba** [comment]
Ok, thanks for the quickly response :)
